hack/arktos-cni.rc

```diff
@@ -29,6 +29,9 @@ setup_cni_conf() {
     if [ -s ${conf_target_file} ]; then
       echo "${conf_target_file} already exists; keep it."
       return
+    else
+      echo "deleting other existing conf files"
```

Comment:
Why do we need to do this?

Reply:
We have removed the code and changed the location for CNI conf backup location to /var/backups/ as suggested.